### DISCUSSION TITLE
Pin Link, categorize or add tags

### DISCUSSION THREAD

**Discussion by ItsNoted** [Just-Moh-it/Pckd#42] [Version 2]
Allow users to create categories or tags 🏷 for their links 🔗. Useful for those using this as a bookmark manager. 📑

**Comment by Just-Moh-it** [comment]
Have already been thinking about it since I use Pckd daily, and felt the need for better categorization. I'll create a GitHub project spreadsheet on this repo and add it there...

**Comment by ItsNoted** [comment]
Thanks for the quick replies. Just sent a donation 🥇 👍 💯

**Comment by Just-Moh-it** [comment]
Thank you so much... Things like these make me super-excited about bringing something new to the table... Can't express how over-joyed I am 🤩😄 Will make sure the money goes towards the required human resources for making Pckd and other projects better 😊
